src/libsync/syncengine.h

```diff
@@ -72,7 +73,9 @@ class OWNCLOUDSYNC_EXPORT SyncEngine : public QObject
      * -1 means infinite
      */
     void setNewBigFolderSizeLimit(qint64 limit) { _newBigFolderSizeLimit = limit; }
+    void setIgnoreHiddenFiles(bool ignore) { _csync_ctx->ignore_hidden_files = ignore; }
 
+    ExcludedFiles &excludedFiles() { return *_excludedFiles; }
```

Comment:
Would you prefer a pointer? The reference makes the non-nullability clear though.